src/DynamoCoreWpf/ViewModels/Core/SimpleViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel;
+
+namespace Dynamo.ViewModels
+{
+    /// <summary>
+    /// Dead simple view model for when you only need to have a different display name and underlying value.
+    /// </summary>
+    /// <typeparam name="T">The type of your value</typeparam>
+    public class SimpleViewModel<T> : ViewModelBase, INotifyPropertyChanged
```

Comment:
do you expect this to ever grow to contain more properties?

Maybe we can call it `NameValuePairViewModel`?

Reply:
Ok